include/psxsdk/libgte.h

```diff
@@ -18,11 +18,27 @@ typedef struct {
     short pad;        // System reserved
 } SVECTOR;            // size = 0x8
 
+typedef struct {
+    /* 0x00 */ SVECTOR* v0;
+    /* 0x04 */ SVECTOR* v1;
+    /* 0x08 */ SVECTOR* v2;
+    /* 0x0C */ SVECTOR* v3;
+} SVEC4; // size = 0x10
```

Comment:
Is this a libgte type or something you made? If you made it then it should probably be in game-specific header.